components/sidebar/sidebar.jsx

```diff
@@ -237,26 +241,18 @@ export default class Sidebar extends React.PureComponent {
     }
 
     setBadgesActiveAndFavicon() {
+        const link = document.querySelector('link[rel="icon"]');
+
+        if (!link) {
```

Comment:
Maybe we could check this inside `this.updateFavicon` since that's where we actually modify those links?

Reply:
Well, If I move this into `this.updateFavicon`, `this.badgesActive` will be updated even if there are no favicon links in the document, which can cause inconsistency between `this.badgesActive` and what favicons are really representing.
So I put that code before updating `this.badgesActive`.

Reply:
That's a good point. I don't think we can actually get into a case where those links don't exist, but it doesn't hurt to keep this around